ingestion/src/metadata/ingestion/source/dashboard/looker/metadata.py

```diff
@@ -190,12 +274,31 @@ def parser(self, all_lookml_models: Sequence[LookmlModel]) -> None:
         if self.repository_credentials:
             all_projects: Set[str] = {model.project_name for model in all_lookml_models}
             self._project_parsers: Dict[str, LkmlParser] = {
-                project_name: LkmlParser(
-                    reader=self.reader(
-                        credentials=self.get_lookml_project_credentials(
-                            project_name=project_name
-                        )
-                    )
+                # project_name: LkmlParser(
+                #     # reader=self.reader(
+                #     #     credentials=self.get_lookml_project_credentials(
+                #     #         project_name=project_name
+                #     #     )
+                #     # )
+                #     reader=self.reader(Path(self._main_lookml_repo.path)),
+                #     remote_readers={
```

Comment:
if this code is not relevant anymore we can delete the comment block

Reply:
I forgot to ingtegrate with bitbucket, do you think that it can work with both github & bitbucket?

Reply:
if the lib you are using to clone handles both then it should be fine, but we should have support for both sources yes